MAUI/Cards/Customization.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: Customization in MAUI Cards control | Syncfusion
+description: Learn about Customization support in Syncfusion Essential Studio MAUI Cards control, its elements and more.
+platform: MAUI
+control: Cards
+documentation: ug
+---
+
+# Customization in MAUI Cards
+
+### ShowSwipedCard
+
+Enabling the `ShowSwipedCard` option allows the swiped cards to be displayed at the edge of the card layout.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<cards:SfCardLayout ShowSwipedCard="true">
+</cards:SfCardLayout>
+ 
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfCardLayout cardLayout = new SfCardLayout()
+{
+  ShowSwipedCard = true
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![ShowSwipedCard in MAUI SfCardLayout](cardlayout-customization_images/showswipedcard.png)
+
+### VisibleIndex
+
+The `VisibleIndex` is utilized when providing the index of the card that is intended to be displayed at the front of the card layout.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<cards:SfCardLayout VisibleIndex="1">
+</cards:SfCardLayout>
+ 
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfCardLayout cardLayout = new SfCardLayout()
+{
+  VisibleIndex = 1
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![VisibleCardIndex in MAUI SfCardLayout](cardlayout-customization_images/showswipedcard.png)
+
+### SwipeDirection
+
+The `SwipeDirection` property specifies the direction of swiping, which can be left, right, top, or bottom. Default value of swipe direction is right.
```

Comment:
The `SwipeDirection` property specifies the direction of swiping, which can be left, right, top, or bottom. The default value of swipe direction is right.